macos-example/package.json

```diff
@@ -12,7 +12,7 @@
     "react": "18.2.0",
     "react-native": "0.73.4",
     "react-native-macos": "^0.73.0-0",
-    "react-native-reanimated": "3.9.0-nightly-20240421-474f6983d",
+    "react-native-reanimated": "3.13.0",
```

Comment:
Maybe we could wait with that update (macos only) for the `react-native-macos@0.74`+? 
```
[!] Invalid `RNReanimated.podspec` file: [Reanimated] Outdated version of React Native for New Architecture. Reanimated 3.13.0 supports the New Architecture on React Native 0.74.0+. See https://docs.swmansion.com/react-native-reanimated/docs/guides/troubleshooting#outdated-version-of-react-native-for-new-architecture for more information.
```